client/src/components/CreateListForm.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { InputName } from './InputName';
+import { InputDescription } from './InputDescription';
+import { ButtonSubmit } from './ButtonSubmit';
+import { ButtonCancel } from './ButtonCancel';
+
+export function CreateListForm(props) {
+  return (
+    <form onSubmit={props.handleSubmit}>
+      <div className="p-m-6 p-d-flex p-jc-center">
+        <div className="card">
+          <h2>Create New List</h2>
+          <label>
+            <h3>Name</h3>
+            <InputName
+              state={props.state}
+              handleSetState={props.handleSetState}
+            />
+          </label>
+
+          <label>
+            <h3>Description</h3>
+            <InputDescription
+              state={props.state}
```

Comment:
We can make it clear here, what kind of state happens here? It is an input description

`state={props.description}`

`handleSetState={props.onDescriptionChange}`